### ISSUE TITLE
Add emoji-font requirement to documentation for unicode build and coordinate w/ maintainers

### ISSUE THREAD

**Issue by jaimet** [patroclos/PAmix#65]
Hi.

Apologies for the weird question: what font are you using in the [screenshot](http://i.imgur.com/NuzrAXZ.gif) ? I am asking as I can't find anything monospace with both U+1F512 ("LOCK") and U+1F507 ("SPEAKER WITH CANCELLATION STROKE") glyphs, but clearly you have!

Thank you.

**Comment by patroclos** [comment]
I think that was Hack

**Comment by jaimet** [comment]
Now that you mention it, I can see that the "0" (zero) is definitely from Hack (Hack's "0" is very noticeable). Hack contains neither U+1F507 nor U+1F512 however, so you must be getting some font-config substition going on, and seeing those glyphs from another font on your system. No worries - I'll just recompile without those characters. Thanks again.

**Comment by patroclos** [comment]
good point, idk anymore about that setup, but I'm also using noto-fonts-emoji, that seems to do the trick

**Comment by jaimet** [comment]
Here's the (now deprecated) NotoEmoji glyph:
![NotoEmoji](https://user-images.githubusercontent.com/8218065/93390478-a7dec000-f865-11ea-816d-bd1690af3f6c.png)
and here's the (not deprecated) NotoColorEmoji glyph:
![NotoColorEmoji](https://user-images.githubusercontent.com/8218065/93395336-a0bbb000-f86d-11ea-99f5-8757b26b435d.png)
so I don't think it's either of those. That's not a problem for me, however, as I really do need to compile from master HEAD and I'll go for the non-wide version.

I now understand that those 2 glyphs are in  the "Miscellaneous Symbols and Pictographs" Unicode block so, in order to avoid the following "tofu-bake"
![tofubake](https://user-images.githubusercontent.com/8218065/93395880-9c43c700-f86e-11ea-8cae-341a32f3519c.png)
 the user needs to install an "emoji" font with that Unicode block present (such as NotoColorEmoji). Given that (at least) debian packages "wide unicode" pamix with no "package recommendations" or "package suggestions" to install a suitable font, I'm guessing that some of your users will see the "tofu-bake" version (as I did when I first installed it). What do you think about the idea of mentioning this "font requirement" in the documentation? (If it's in the documentation, debian might even change their package control fields to suggest/recommend a suitable font.)

Just an idea. J

**Comment by CodeMonkey80s** [comment]
Just adding to the conversation I think there should be a command line option to disable custom fonts (emoji) usage all together, so only ASCII (extended) symbols are being used?

**Comment by ghost** [comment]
Late to the party but all these years later I am still struggling to get this to look right. All the emoji capable fonts I've tried show the symbols but then everything else is terrible. I have actually been using Terminus for years now as a decent inbetween but today for some reason everything has changed. Volume indicators are all empty boxes like with most fonts and I'm kinda back to wondering what font will jive. So what font(s) are you guys using to remedy this? For the record I'm down with pure ascii, I don't *need* emoji/symbols, I just would rather not have empty boxes.
